src/RestApi/StoreApi/Schemas/OrderSchema.php

```diff
@@ -0,0 +1,625 @@
+					'total_fees_tax'     => $this->prepare_money_response( $this->get_fee_tax( $object ), wc_get_price_decimals() ),
+					'total_discount'     => $this->prepare_money_response( $object->get_discount_total(), wc_get_price_decimals() ),
+					'total_discount_tax' => $this->prepare_money_response( $object->get_discount_tax(), wc_get_price_decimals() ),
+					'total_shipping'     => $this->prepare_money_response( $object->get_shipping_total(), wc_get_price_decimals() ),
+					'total_shipping_tax' => $this->prepare_money_response( $object->get_shipping_tax(), wc_get_price_decimals() ),
+					'total_price'        => $this->prepare_money_response( $object->get_total(), wc_get_price_decimals() ),
+					'total_tax'          => $this->prepare_money_response( $object->get_total_tax(), wc_get_price_decimals() ),
+					'tax_lines'          => $this->get_tax_lines( $object ),
+				]
+			),
+		];
+	}
+
+	/**
+	 * Removes the wc- prefix from order statuses.
+	 *
+	 * @param string $status Status from the order.
+	 * @return string
+	 */
+	protected function remove_status_prefix( $status ) {
+		return 'wc-' === substr( $status, 0, 3 ) ? substr( $status, 3 ) : $status;
+	}
+
+	/**
+	 * Get event dates from an order, formatting both local and GMT values.
+	 *
+	 * @param \WC_Order $object Order class instance.
+	 * @return array
+	 */
+	protected function get_events( $object ) {
+		$events = [];
+		$props  = [ 'date_created', 'date_modified', 'date_completed', 'date_paid' ];
+
+		foreach ( $props as $prop ) {
+			$datetime                 = $object->{"get_$prop"}();
+			$events[ $prop ]          = wc_rest_prepare_date_response( $datetime, false );
+			$events[ $prop . '_gmt' ] = wc_rest_prepare_date_response( $datetime );
+		}
+
+		return $events;
+	}
+
+	/**
+	 * Get tax lines from the order and format to match schema.
+	 *
+	 * @param \WC_Order $object Order class instance.
+	 * @return array
+	 */
+	protected function get_tax_lines( $object ) {
+		$tax_totals = $object->get_tax_totals();
+		$tax_lines  = [];
+
+		foreach ( $tax_totals as $tax_total ) {
+			$tax_lines[] = array(
+				'name'  => $tax_total->label,
+				'price' => $this->prepare_money_response( $tax_total->amount, wc_get_price_decimals() ),
+			);
+		}
+
+		return $tax_lines;
+	}
+
+	/**
+	 * Get the total amount of tax for line items.
+	 *
+	 * Needed because orders do not hold this total like carts.
+	 *
+	 * @todo This could be added to the orders class to match the cart class.
+	 *
+	 * @param \WC_Order $object Order class instance.
+	 * @return float
+	 */
+	protected function get_subtotal_tax( $object ) {
+		$total = 0;
+		foreach ( $object->get_items() as $item ) {
+			$total += $item->get_subtotal_tax();
+		}
+		return $total;
+	}
+	/**
+	 * Get the total amount of fees.
+	 *
+	 * Needed because orders do not hold this total like carts.
+	 *
+	 * @todo This could be added to the orders class to match the cart class.
```

Comment:
When you say orders class in this todo here and elsewhere, are you referring to the `\WC_Order` class?